explainer.md

```diff
@@ -193,6 +190,13 @@ active service worker:
 6. If [[*handlePending*]] is rejected, proceed with the normal navigation
    algorithm. (If it is fulfilled, do not continue with the navigation.)
 
+The user agent **MAY** go through the `launch` flow after going through the
```

Comment:
Since this is an explainer, not spec, I think we could go into some more specifics here (even at the risk of being Chrome-specific), perhaps with screenshots.

To give readers an idea of what this is all about, and why we want to prompt the user first.

Reply:
Specifics added